ecosystem/indexer/src/models/ledger_info.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::schema::ledger_infos;
+
+#[derive(Debug, Identifiable, Insertable, Queryable)]
+#[diesel(table_name = "ledger_infos")]
+#[primary_key(chain_id)]
+pub struct LedgerInfo {
+    pub chain_id: i64,
```

Comment:
So we only store the chain ID? I figure in that case, we don't intend to repeatedly fetch the ledger info? If we do, should we consider storing stuff like ledger version as well (the other stuff returned by `/`).

Reply:
Ledger version is just the highest version which isnt necessary